enterprise_knowledge_retriever/src/document_retrieval.py

```diff
@@ -239,54 +332,30 @@ def init_retriever(self, vectorstore: Any) -> None:
                 },
             )
 
-    def get_qa_retrieval_chain(self) -> RetrievalQAChain:
+    def get_qa_retrieval_chain(self, conversational: bool = False) -> RetrievalQAChain:
```

Comment:
Can we add 'conversational' as an input arg in config.yaml that users can control? Thanks

Reply:
added under retrieval, given retrieval is done over reformulated or original query